File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-05-15-preview/examples-js/CosmosDBManagedCassandraClusterDelete.js
const { CosmosDBManagementClient } = require("@azure/arm-cosmosdb");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Deletes a managed Cassandra cluster.
 *
 * @summary Deletes a managed Cassandra cluster.
 * x-ms-original-file: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-05-15-preview/examples/CosmosDBManagedCassandraClusterDelete.json
 */
async function cosmosDbManagedCassandraClusterDelete() {
  const subscriptionId = "subid";
  const resourceGroupName = "cassandra-prod-rg";
  const clusterName = "cassandra-prod";
  const credential = new DefaultAzureCredential();
  const client = new CosmosDBManagementClient(credential, subscriptionId);
  const result = await client.cassandraClusters.beginDeleteAndWait(resourceGroupName, clusterName);
  console.log(result);
}

cosmosDbManagedCassandraClusterDelete().catch(console.error);

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-08-15/examples-java/CosmosDBManagedCassandraClusterDelete.java
import com.azure.core.util.Context;

/** Samples for CassandraClusters Delete. */
public final class Main {
    /*
     * x-ms-original-file: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-08-15/examples/CosmosDBManagedCassandraClusterDelete.json
     */
    /**
     * Sample code: CosmosDBManagedCassandraClusterDelete.
     *
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void cosmosDBManagedCassandraClusterDelete(com.azure.resourcemanager.AzureResourceManager azure) {
        azure
            .cosmosDBAccounts()
            .manager()
            .serviceClient()
            .getCassandraClusters()
            .delete("cassandra-prod-rg", "cassandra-prod", Context.NONE);
    }
}
